networking/configure-syscontrols-interface-tuning-cni.adoc

```diff
@@ -1,26 +1,25 @@
 :_mod-docs-content-type: ASSEMBLY
-:context: set-networkinterface-sysctls
-[id="nodes-setting-interface-level-network-sysctls"]
-= Configuring interface-level network sysctls
+[id="configure-syscontrols-interface-tuning-cni"]
+= Configuring system controls and interface attributes using the tuning plugin
 include::_attributes/common-attributes.adoc[]
+:context: configure-syscontrols-interface-tuning-cni
 
 toc::[]
 
 In Linux, sysctl allows an administrator to modify kernel parameters at runtime. You can modify interface-level network sysctls using the tuning Container Network Interface (CNI) meta plugin. The tuning CNI meta plugin operates in a chain with a main CNI plugin as illustrated.
 
 image::264_OpenShift_CNI_plugin_chain_0722.png[CNI plugin]
 
-The main CNI plugin assigns the interface and passes this to the tuning CNI meta plugin at runtime. You can change some sysctls and several interface attributes (promiscuous mode, all-multicast mode, MTU, and MAC address) in the network namespace by using the tuning CNI meta plugin.  In the tuning CNI meta plugin configuration, the interface name is represented by the `IFNAME` token, and is replaced with the actual name of the interface at runtime.
-
-[NOTE]
-====
-In {product-title}, the tuning CNI meta plugin only supports changing interface-level network sysctls.
-====
+The main CNI plugin assigns the interface and passes this to the tuning CNI meta plugin at runtime. You can change some sysctls and several interface attributes (promiscuous mode, all-multicast mode, MTU, and MAC address) in the network namespace by using the tuning CNI meta plugin.
```

Comment:
```suggestion
The main CNI plugin assigns the interface and passes this interface[?] to the tuning CNI meta plugin at runtime. You can change some sysctls and several interface attributes such as promiscuous mode, all-multicast mode, MTU, and MAC address in the network namespace by using the tuning CNI meta plugin.
```
- `this` what? The pronoun needs a reference for clarity.
- [avoid parentheticals](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses)